bst.py

```diff
@@ -113,14 +120,65 @@ def _get_dot(self, current):
             yield "\tnull%s [shape=point];" % r
             yield "\t%s -> null%s;" % (current, r)
 
+    def in_order(self, current='start'):
+        """
+        Generator that traverses the binary tree in order.
+        """
+        if current == 'start':
+            current = self.top
```

Comment:
What if your BST is containing strings, like perhaps one might if you were building a dictionary for spell-checking or something.  What if the word `start` is in your dictionary?